src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.test.tsx

```diff
@@ -138,7 +138,7 @@ describe('<GeneOverview />', () => {
       expect(geneDetailsElement?.textContent).toBe(stableId);
     });
 
-    it('does not render gene name section if gene name is not available', () => {
+    it('shows "None" message for gene without gene name', () => {
```

Comment:
Not insisting upon this change, but wondering whether the description of the test might benefit from a little bit less detail?

E.g.: `it('shows that the gene does not have a name')` and `it('shows that the gene does not have synonyms')`?

Reply:
In my head the test was to check if it is showing the message "None" and hence the description. Because before it was showing No synonyms instead but we had to change this. I wanted to put the emphasis on "None" since it looks like that the right word. Is that a good enough reason to not change it :) if not i am happy to change it.

Reply:
The word "None", as the phrase "no synonyms" before it (mea culpa), is a very specific implementation detail, which we are already checking in the body of the test. It's generally advisable to write test descriptions at somewhat higher level than the actual implementation details that are being tested.

Reply:
Makes sense @azangru i have pushed the changes.